mathemagic.py

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env python3
+'''
+This script computes the actual sum of the five "random" four-digit numbers 
+to cross-check with the predicted sum and display the result to the user.
+'''
+
+#A variable is declared to store the user's name and input number
```

Comment:
All comments should have a space between the comment sign and the actual comment.